interface/main/tabs/js/tabs_view_model.js

```diff
@@ -131,7 +131,7 @@ function navigateTab(url,name,afterLoadFunction,loading_label='')
         if(typeof afterLoadFunction !== 'function'){
             $( "body" ).off( "load", "iframe[name='"+name+"']");
         } else {
-            $("iframe[name='"+name+"']").on('load', function () {
+            $("iframe[name='"+name+"']").one('load', function () {
```

Comment:
dang, one vowel bug fix. that deserves OpenEMR fans!
![2DV](https://user-images.githubusercontent.com/278968/103489407-30f41580-4dc9-11eb-9215-80fe9647a8c3.gif)

Reply:
aw shucks! for me.
I happenstance across another to fix. when opening document from encounter history then return from documents, we open a new dashboard tab....

Reply:
Going to have to wait to fix document return from encounter as it uses browser history.
Bringing in